control-plane/subcommand/inject-connect/command.go

```diff
@@ -222,6 +224,8 @@ func (c *Command) init() {
 			"%q, %q, %q, and %q.", zapcore.DebugLevel.String(), zapcore.InfoLevel.String(), zapcore.WarnLevel.String(), zapcore.ErrorLevel.String()))
 	c.flagSet.BoolVar(&c.flagLogJSON, "log-json", false,
 		"Enable or disable JSON output format for logging.")
+	c.flagSet.BoolVar(&c.flagResourceAPIs, "resource-apis", false,
```

Comment:
[NIT] make it consistent with other flags.
```suggestion
	c.flagSet.BoolVar(&c.flagResourceAPIs, "enable-resource-apis", false,
```